modin/conftest.py

```diff
@@ -342,6 +348,133 @@ def make_csv_file():
     teardown_test_files(filenames)
 
 
+def create_and_call(filenames, filename, force, nrows, ncols, func: str, func_kw=None):
```

Comment:
It's a little bit confusing function name, will be great if docstring describing it possible purpose will be added.

Reply:
Renamed and moved this function inside `make_default_file`, as it is very narrowly focused. Do I need to add documentation?

Reply:
Yes, i think docstring with note will be fine (something describing what should be created and why then call `func`).